lib/imagec/docker.go

```diff
@@ -100,13 +100,22 @@ func LearnRegistryURL(options Options) (string, error) {
 		return registry, nil
 	}
 
-	// first try https
 	log.Debugf("Trying https scheme")
```

Comment:
Might want to update this debug message to be a bit more useful.  I.e. echoing the url (if not echoed elsewhere and) whether insecure is set.